src/plugins/tables/reports/pre-a11y-1.json

```
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/tables/tables-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Tables",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "wb-tables"
```

I think you means the CSS class selector

```suggestion
            "@value": ".wb-tables"
```